master/getting-started/kubernetes/tutorials/app-layer-policy/index.md

```diff
@@ -0,0 +1,298 @@
+
+    curl -k https://database:2379/v2/keys/accounts/519940/balance -d value="10000.00" -XPUT --key /etc/certs/key.pem --cert /etc/certs/cert-chain.pem
+
+Unlike when we did this with the customer web pod, we do not have the Envoy to handle encryption, so
+we have to pass an `https` URL, the `--key` and `--cert` parameters to `curl` to do the
+cryptography.
+
+Return to your web browser and refresh to confirm the new balance.
+
+#### Policy
+
+We can mitigate both of the above deficiencies with a Calico policy.
+
+    wget {{site.url}}/{{page.version}}/getting-started/kubernetes/tutorials/app-layer-policy/manifests/30-policy.yaml
+    calicoctl create -f 30-policy.yaml
+
+> **Note**: You can also
+> [view the manifest in your browser](manifests/30-policy.yaml){:target="_blank"}.
+{: .alert .alert-info}
+
+Let's examine this policy piece by piece.  It consists of 3 policy objects, one for each microservice.
+
+    apiVersion: projectcalico.org/v3
+    kind: GlobalNetworkPolicy
+    metadata:
+      name: customer
+    spec:
+      selector: app == 'customer'
+      ingress:
+        - action: Allow
+          http:
+            methods: ["GET"]
+      egress:
+        - action: Allow
+
+This policy protects the customer web app.  Since this application is customer facing, we do not
+restrict what can communicate with it.  We do, however, restrict that only HTTP `GET` requests are
+allowed.
+
+    apiVersion: projectcalico.org/v3
+    kind: GlobalNetworkPolicy
+    metadata:
+      name: summary
+    spec:
+      selector: app == 'summary'
+      ingress:
+        - action: Allow
+          source:
+            serviceAccounts:
+              names: ["customer"]
+      egress:
+        - action: Allow
+
+The second policy protects the account summary microservice.  We know the only consumer of this
+service is the customer web app, so we restrict the source of incoming connections to the service
+account for the customer web app.
+
+    apiVersion: projectcalico.org/v3
+    kind: GlobalNetworkPolicy
+    metadata:
+      name: database
+    spec:
+      selector: app == 'database'
+      ingress:
+        - action: Allow
+          source:
+            serviceAccounts:
+              names: ["summary"]
+      egress:
+        - action: Allow
+
+The third policy protects the database.  Only the summary microservice should have direct access to
+the database.
+
+Let's verify our policy is working as intended.  First, return to your browser and refresh, to
+ensure policy enforcement has not broken the application.
+
+Next, return to the customer web app.  Recall that we simulated an attacker gaining control of that
+pod by `exec`ing into it.
+
+    kubectl exec -ti customer-<fill in pod ID> -c customer bash
+
+Repeat our attempt to access the database.
+
+    curl http://database:2379/v2/keys?recursive=true
+
+This time we should get a 403 Forbidden response (we have left out the JSON formatting because we do
```

Comment:
I would like to see 403 Forbidden in backticks. Also, I would like to see the paranthetical aside instead as a sentence...as the first sentence.